src/char/char_clif.c

```diff
@@ -885,61 +888,143 @@ int chclif_parse_charselect(int fd, struct char_session_data* sd,uint32 ipl){
 	return 1;
 }
 
+enum e_charclif_error {							//tested with 0x6e in 20150914
+	ERROR_SAMENAME_EXIST = 0x0,					//Character name already exist
+	ERROR_LIMIT_AGE = 0x1,						//You are underaged
+	//ERROR_LIMIT_CHAR_DELETE = 0x2,			
+	ERROR_INVALID_SYMBOLS = 0x2,				//Symbols in Character Names are forbidden
+	ERROR_MAKECHAR_INVALID_SLOT = 0x3,			//You are not elegible to open the Character Slot
+	ERROR_NEED_PREMIUM_SERVICE = 0xb,			//This service is only available for premium users
+	ERROR_NONPERMISSION_NAME = 0xc,				//Failed to move char slot
```

Comment:
If the player failed to move the character to another slot why is the name `ERROR_NONPERMISSION_NAME`? That doesn't seem like it fits this use case. Something like `MAKECHAR_SLOT_MOVE_FAIL` or something along those lines.